be/src/Project/Services/Analytics/Analytics.Api/v1/Controllers/SalesAnalyticsController.cs

```diff
@@ -0,0 +1,168 @@
+            {
+                Language = CultureInfo.CurrentCulture.Name,
+                OrganisationId = GuidHelper.ParseNullable(sellerClaim?.Value),
+                Username = this.User.Claims.FirstOrDefault(x => x.Type == ClaimTypes.Email)?.Value,
+                IsSeller = this.User.IsInRole("Seller")
+            };
+
+            var validator = new GetTopSalesAnalyticsModelValidator();
+            var validationResult = await validator.ValidateAsync(serviceModel);
+
+            if (validationResult.IsValid)
+            {
+                var topSalesProducts = await this.salesService.GetTopSalesProductsAnalyticsAsync(serviceModel);
+
+                if (topSalesProducts is not null)
+                {
+                    var response = topSalesProducts.Select(x => new TopSalesProductsAnalyticsResponseModel
+                    {
+                        ProductId = x.ProductId,
+                        ProductSku = x.ProductSku,
+                        ProductName = x.ProductName,
+                        Ean = x.Ean,
+                        Quantity = x.Quantity
+                    });
+
+                    return this.StatusCode((int)HttpStatusCode.OK, response);
+                }
+            }
+
+            throw new CustomException(string.Join(ErrorConstants.ErrorMessagesSeparator, validationResult.Errors.Select(x => x.ErrorMessage)), (int)HttpStatusCode.UnprocessableEntity);
+        }
+
+        /// <summary>
+        /// Creates sales facts
+        /// </summary>
+        /// <param name="request">The model.</param>
+        /// <returns>OK.</returns>
+        [HttpPost, MapToApiVersion("1.0")]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.UnprocessableEntity)]
+        public async Task<IActionResult> Save(SalesAnalyticsRequestModel request)
+        {
+            var sellerClaim = this.User.Claims.FirstOrDefault(x => x.Type == AccountConstants.Claims.OrganisationIdClaim);
+
+            var serviceModel = new CreateSalesAnalyticsServiceModel
+            {
+                Language = CultureInfo.CurrentCulture.Name,
+                OrganisationId = GuidHelper.ParseNullable(sellerClaim?.Value),
+                Username = this.User.Claims.FirstOrDefault(x => x.Type == ClaimTypes.Email)?.Value,
+                SalesAnalyticsItems = request.SalesAnalyticsItems.Select(x => new CreateSalesAnalyticsItemServiceModel
+                {
+                    ClientId = x.ClientId,
+                    ClientName = x.ClientName,
+                    Email = x.Email,
+                    ProductId = x.ProductId,
+                    ProductName = x.ProductName,
+                    ProductSku = x.ProductSku,
+                    Ean = x.Ean,
+                    IsOutlet = x.IsOutlet,
+                    IsStock = x.IsStock,
+                    Quantity = x.Quantity,
+                    Country = x.Country,
```

Comment:
**suggestion:** let's merge the develop branch to this one. Let's use country id and introduce the dimension country with translation for name.